pkg/controller/miganalytic/miganalytics_controller.go

```diff
@@ -236,8 +236,10 @@ func (r *ReconcileMigAnalytic) analyze(analytic *migapi.MigAnalytic) error {
 		return liberr.Wrap(err)
 	}
 
-	analytic.Status.Analytics.Plan = plan.Name
+	nodeToPVMap := make(map[string][]MigAnalyticPersistentVolumeDetails)
 
+	analytic.Status.Analytics.Plan = plan.Name
+	analytic.Status.Analytics.Namespaces = make([]migapi.MigAnalyticNamespace, 0)
```

Comment:
do we know the namespaces length already? can we seen it with length that is not 0?

Reply:
FTR its a NIT, not a merge blocker